go.mod

```diff
@@ -91,7 +94,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.16.0
 	github.com/stretchr/testify v1.8.4
-	github.com/testcontainers/testcontainers-go v0.23.0
+	github.com/testcontainers/testcontainers-go v0.25.0
```

Comment:
Hello @simar7 !

I investigated errors with integration tests.
Looks like problem is with `testcontainers-go`versions `v0.25.0` and `v0.26.0` -https://github.com/testcontainers/testcontainers-go/issues/1782.

Do you need to update this dependency now or we can this be done after fix from `testcontainers-go`?

Reply:
Hello @DmitriyLewen - I see thanks for looking into it. I will try to go back to v0.23.0 to see if tests are green.